### ISSUE TITLE
Hello, could you tell me the proportion of covid-19 training set, verification set and test set? Is it random?

### ISSUE THREAD

**Issue by qianghuochangge** [HUANGLIZI/LViT#6]
Hello, could you tell me the proportion of covid-19 training set, verification set and test set? Is it random?

**Comment by HUANGLIZI** [comment]
please follow the official proportion of the QaTa-COV19 dataset.

**Comment by qianghuochangge** [comment]
However, the validation set is not included in the official data set of QaTa-COV19. How did you deal with it?

**Comment by HUANGLIZI** [comment]
Please email me by your official email.

**Comment by qianghuochangge** [comment]
Hello, I have sent my question to your email, please pay attention to check, I hope to get your reply in time, thank you
